src/rebar_prv_compile.erl

```diff
@@ -303,6 +309,7 @@ build_rebar3_apps(DAGs, Apps, State) ->
     LastDAG = lists:last(DAGs),
     %% we actually need to compile each DAG one after the other to prevent
     %% issues where a .yrl file that generates a .erl file gets to be seen.
+    ?INFO("Analyzing applications", []),
```

Comment:
Should this be debug? I'm fine with info but maybe add `...` to the end. I know we haven't been consistent but I like that for when a task is in the process and not a message about something that completed.

Reply:
I mostly added it because the new analysis phase severely delays "compiling <app>..." messages, and it makes it look like rebar3 is verifying dependencies even when there are none.